### PR TITLE
Generate site index page using sections content

### PR DESCRIPTION
**Pull request by amuraru**
## Description

Use site sections summary to auto-generate index page.

sample rendering:
image

### CODE REVIEW

File: layouts/_default/summary.html [hunk 1]
```diff
@@ -0,0 +1,9 @@
+<span style="text-align: center;">
```
Comment (alexander-falca) [inline layouts/_default/summary.html:1]: wdyt about moving this to `partials` folder?

**Review by alexander-falca — APPROVED** [review]
nice one!
see a minor comment and then :+1:

File: layouts/_default/summary.html (hunk 1 above)
Comment (amuraru) [reply layouts/_default/summary.html:1]: not sure I get this - you want me to move the whole summary as a partial or just the span here?

Comment (alexander-falca) [reply layouts/_default/summary.html:1]: I meant moving the whole summary :)

**Comment by amuraru** [comment]
I would keep it as a layout. Took the recipe actually from https://gohugo.io/templates/views/#example-content-view-inside-a-list

Lmk.

**Comment by alexander-falca** [comment]
I don't have strong preference here, so let's keep then as is :)

**Pull request merged**